### PR TITLE
Cardss

### PR DESCRIPTION
**Pull request by sahiba0915**
Added Card Component

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+}
+
+.hero-img {
+  width: 50%;
+}
+
+/* buttons */
+.hero-btns {
+  display: flex;
+  flex-direction: row;
```
Comment (haarsh24) [inline styles.css:116]: you can also declare these properties in util. as these can be used in multiple places.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+.sidebar-list li a {
+  text-decoration: none;
+  font-size: 1.5rem;
+  color: var(--secondary-text);
+}
+
+.sidebar-list li a:hover {
+  color: #000000;
+}
```
Comment (haarsh24) [inline styles.css:190]: Declare this color and use it here.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+  margin-left: 2rem;
+  padding: 1.5rem;
+  padding-bottom: 2.5rem;
+  font-family: var(--primary-font);
+}
+.intro-heading {
+  margin-top: 1.5rem;
+  padding-bottom: 1rem;
+  font-size: 3rem;
+  color: black;
```
Comment (haarsh24) [inline styles.css:205]: This color also

File: component/avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
+.avatar-container {
+  display: flex;
+  flex-direction: column;
```
Comment (haarsh24) [inline component/avatar/avatar.css:3]: declare these properties also , and use it . same as flex-row.

**Review by haarsh24 — CHANGES_REQUESTED** [review]
Declare color and basic flex properties in util.css and use them in the code.
Rest all looks good.

File: component/alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  color: var(--secondary-text);
+}
+
+.alert-title {
+  font-size: 2rem;
+  padding-top: 1.5rem;
+  color: black;
+}
+.alert {
+  display: flex;
```
Comment (saifudinbadani) [inline component/alert/alert.css:28]: if using flex property repeatedly, maybe make that a class.

**Review by saifudinbadani — APPROVED** [review]
Otherwise, looks good to merge.

**Comment by poojakelkar** [comment]
looks good to merge.

**Comment by sahiba0915** [comment]
Merging Card Component

File: styles.css (hunk 1 above)
Comment (sahiba0915) [reply styles.css:116]: done

File: styles.css (hunk 2 above)
Comment (sahiba0915) [reply styles.css:190]: done

File: styles.css (hunk 3 above)
Comment (sahiba0915) [reply styles.css:205]: done

File: component/avatar/avatar.css (hunk 4 above)
Comment (sahiba0915) [reply component/avatar/avatar.css:3]: done

File: component/alert/alert.css (hunk 5 above)
Comment (sahiba0915) [reply component/alert/alert.css:28]: done

**Pull request merged**